app/models/concerns/spotlight/translatables.rb

```diff
@@ -16,7 +16,13 @@ def translates(*attr_names)
       # and defaults to the ActiveRecord value
       def define_translated_attr_reader(attr_name)
         define_method(:"#{attr_name}") do
-          I18n.translate(attr_name, scope: slug, default: attr_translation(attr_name))
+          send("translated_#{attr_name}", default: [attr_translation(attr_name), ''])
+        end
+
+        define_method(:"translated_#{attr_name}") do |default: [], **options|
```

Comment:
Would it be possible to add a comment here as to what this dynamically defined method is doing?  I'm a bit miffed at the `default.prepend` here which seems like it would duplicate the value (since `attr_translation(attr_name)` is prepended and is the default) under the case that the current locale is the default locale.

I'm assuming I'm miss-reading/understanding the intent of the code, but perhaps a comment could clear that up.  I would also be happy to have it explained here in the PR and realize that I'm just confusing myself unnecessarily.

Reply:
Yep, you got it. I've suggested a comment about why it appears doubled.